marshal/message.go

```diff
@@ -45,6 +45,10 @@ const (
 	msgTypeClaimingMessages   msgType = 3
 	msgLengthClaimingMessages int     = msgLengthBase + 1
 	idxCMProposedLastOffset   int     = idxBaseEnd + 1
+
+	msgTypeReleaseGroup   msgType = 4
+	msgLengthReleaseGroup int     = msgLengthBase + 1
+	idxRGAdminClientID    int     = idxBaseEnd + 1
```

Comment:
The message already contains a client ID/instance ID in the base message which is the admin's information. We  don't need to introduce another one, do we?